include/seqan3/range/view/take_until.hpp

```diff
@@ -109,7 +106,7 @@ class view_take_until : public ranges::view_interface<view_take_until<urng_t, fu
         {}
 
         //!\brief Constructor that delegates to the CRTP layer and initialises the callable.
-        iterator_type(base_base_t it, fun_ref_t _fun) :
+        iterator_type(base_base_t it, fun_ref_t _fun, sentinel_type) :
```

Comment:
why does the constructor need a tag parameter of a type that is not a template argument? :eyes: 
EDIT: 
```suggestion
        iterator_type(base_base_t it, fun_ref_t _fun, sentinel_type /*for comppatibility with iterator_type_consume_input*/) :
```